PlatformIO/src/StateMachine.hpp

```diff
@@ -219,6 +223,15 @@ class WifiDisconnected : public StateMachine
     Serial.println("Enter WifiDisconnected");
     Serial.printf("Total heap: %d\r\n", ESP.getHeapSize());
     Serial.printf("Free heap: %d\r\n", ESP.getFreeHeap());
+
+
+#if ESP_TYPE == ESP32_POE_ISO
+
+    WiFi.onEvent(WiFiEvent);
```

Comment:
Is WiFiEvent called? I do no know the ETH code, so I assume yes :)

Reply:
Yes, it is a little odd, but the Ethernet component reuses the WiFi API.